.github/workflows/deploy.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to NPM
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
```
release:
  types:
    - published
```

Reply:
Small change to deploy on a published release. Below comment restricts it further to the main branch.